src/transformState.js

```diff
@@ -6,6 +6,21 @@
  */
 function transformState(state, actions) {
   // write code here
+  for (const act of actions) {
+    if (act.type === 'addProperties') {
+      Object.assign(state, act.extraData);
+    } else if (act.type === 'removeProperties') {
+      for (const i of act.keysToRemove) {
+        delete state[i];
+      }
+    } else {
+      for (const key in state) {
+        delete state[key];
+      }
+    }
```

Comment:
It appears to me that if the user enters a random command as a second argument your function will delete all the data. This is not cool. It should delete only if 'clear' is passed.